src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -1894,32 +1894,46 @@ subroutine atm_compute_vert_imp_coefs_work(nCells, moist_start, moist_end, dts,
       real (kind=RKIND) :: dtseps, c2, qtotal, rcv
       real (kind=RKIND), dimension( nVertLevels ) :: b_tri, c_tri
 
+      MPAS_ACC_TIMER_START('atm_compute_vert_imp_coefs_work [ACC_data_xfer]')
+      !$acc enter data copyin(rdzw, zz, fzm, fzp, rdzu, cqw, p, t, qtot, rb, &
+      !$acc                   rtb, rt, pb)
```

Comment:
Some of the fields here are invariant (across timesteps) and should be handled in the `mpas_atm_dynamics_{init,finalize}` routines instead.

Generally these invariant fields are those contained in the "invariant stream" ([L421-L524 in the atm Registry.xml](https://github.com/MPAS-Dev/MPAS-Model/blob/41e9a3fb8ca6b9250a7405209a5c60996318409f/src/core_atmosphere/Registry.xml#L421-L524)) or those accessed through the "mesh" `mpas_pool` ([L1222-L1516 in the atm Registry.xml](https://github.com/MPAS-Dev/MPAS-Model/blob/41e9a3fb8ca6b9250a7405209a5c60996318409f/src/core_atmosphere/Registry.xml#L1222-L1516))

Reply:
Done. thanks for the catch!